README.md

```diff
@@ -74,6 +74,7 @@ Flags:
       --network uint                ID of the Network
       --private-key string          Private key to use
       --url string                  URL of the node to receive RPC calls (default "ws://localhost:8545")
+      --prepare                     Generate calldata for contract calls
```

Comment:
Maybe also add that it doesn't require private key and it doesn't send transaction